### ISSUE TITLE
Error happens when I try to generate discord accounts/tokens

### ISSUE THREAD

**Issue by cuj** [NightfallGT/Discord-Account-Generator#7]
Hi, when I try to generate discord account/tokens, I get error(s)

so I chose the Normal option, then it opens a chrome window (verison 0.2.149.27) and the website the chrome window is on, is `data:,` then eventually the chrome window crashes, and then the errors that are below, happened, so can you please tell me what did I do wrong/how to fix it? thanks.

```

                             ·▄▄▄▄  ▪  .▄▄ ·  ▄▄·       ▄▄▄  ·▄▄▄▄       ▄▄ • ▄▄▄ . ▐ ▄
                             ██▪ ██ ██ ▐█ ▀. ▐█ ▌▪▪     ▀▄ █·██▪ ██     ▐█ ▀ ▪▀▄.▀·•█▌▐█
                             ▐█· ▐█▌▐█·▄▀▀▀█▄██ ▄▄ ▄█▀▄ ▐▀▀▄ ▐█· ▐█▌    ▄█ ▀█▄▐▀▀▪▄▐█▐▐▌
                             ██. ██ ▐█▌▐█▄▪▐█▐███▌▐█▌.▐▌▐█•█▌██. ██     ▐█▄▪▐█▐█▄▄▌██▐█▌
                             ▀▀▀▀▀• ▀▀▀ ▀▀▀▀ ·▀▀▀  ▀█▄▀▪.▀  ▀▀▀▀▀▀•     ·▀▀▀▀  ▀▀▀ ▀▀ █▪
                                                                    by Nightfall#2512




[*] Proxy used 41.33.191.116:8080
[!] Scraping email.
[*] Scraped fl.et.cher.harv.ey.t.mp@gmail.com
Traceback (most recent call last):
  File "C:\Users\Saturn\Documents\Discord-Account-Generator\discordgenerator.py", line 3
54, in <module>
    worker(random.choice(proxies))
  File "C:\Users\Saturn\Documents\Discord-Account-Generator\discordgenerator.py", line 2
40, in worker
    d = DiscordGen(new_email, username, password, proxy = proxy)
  File "C:\Users\Saturn\Documents\Discord-Account-Generator\discordgenerator.py", line 6
7, in __init__
    self.driver = webdriver.Chrome(options=options, executable_path=r"chromedriver.exe")
  File "C:\Users\Saturn\AppData\Local\Programs\Python\Python39\lib\site-packages\undetected_chromedriver\__init__.py",
line 53, in __new__
    instance.__init__(*args, **kwargs)
  File "C:\Users\Saturn\AppData\Roaming\Python\Python39\site-packages\selenium\webdriver\chrome\webdriver.py", line 76,
 in __init__
    RemoteWebDriver.__init__(
  File "C:\Users\Saturn\AppData\Roaming\Python\Python39\site-packages\selenium\webdriver\remote\webdriver.py", line 157
, in __init__
    self.start_session(capabilities, browser_profile)
  File "C:\Users\Saturn\AppData\Roaming\Python\Python39\site-packages\selenium\webdriver\remote\webdriver.py", line 252
, in start_session
    response = self.execute(Command.NEW_SESSION, parameters)
  File "C:\Users\Saturn\AppData\Roaming\Python\Python39\site-packages\selenium\webdriver\remote\webdriver.py", line 321
, in execute
    self.error_handler.check_response(response)
  File "C:\Users\Saturn\AppData\Roaming\Python\Python39\site-packages\selenium\webdriver\remote\errorhandler.py", line
242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: DevToolsActivePort file doesn't exist
 ```

**Comment by NightfallGT** [comment]
Are you using the normal Chrome web browser?

**Comment by TaprisSugarbell** [comment]
You probably do not have the correct web browser

**Comment by cuj** [comment]
Solved by uninstalling chrome 0.2.149.27, now it works perfectly fine.
